_people/paul_kerensa.mdown

```diff
@@ -0,0 +1,17 @@
+---
+title: Paul Kerensa
+submitted: false
+gender: male
+headshot: dwxQCCF
+careers:
+  - Stand-Up Comedian
+  - Comedy Writer
+  - Actor
+links:
+  - type: Personal Website
+    href: "http://www.paulkerensa.com/"
+  - type: Twitter
+    username: paulkerensa
+---
+
+Since Graduating, Paul has gone on to have an extensive career in both Television and Radio. His Writing credits include the BBC sitcoms Not Going out and Miranda, and he is a regular contributor to BBC Radio 2's Pause for Thought. Paul has won numerous awards for both his writing and performing, including ITV's Take The Mike stand-up award, A British Comedy Award for Miranda and a BAFTA also for Miranda. Paul is a regular up at the Edinburgh Fringe, and performs at both comedy clubs and churches.
```

Comment:
This is going to be very pernickety. You should configure your text editor to automatically put a blank empty line at the end of the file. It's what that symbol is showing (no newline at EOF).

https://robots.thoughtbot.com/no-newline-at-end-of-file